spec/Observable-spec.ts

```diff
@@ -18,7 +18,10 @@ function expectFullObserver(val: any) {
 /** @test {Observable} */
 describe('Observable', () => {
   let originalConfigPromise: any;
-  before(() => originalConfigPromise = config.Promise);
+  before(() => {
```

Comment:
This change is unnecessary for the PR.

Reply:
I had to make this change because of the mocha types update.
The `before` function should return `void`, but here it returns a promise so it failed my build.
If we decide to leave the types change out, then this is indeed unnecessary and the previous comment's resolution will also be this one's.